docs/data/toolpad/concepts/custom-components.md

```diff
@@ -2,11 +2,22 @@
 
 <p class="description">You can bring your own custom components inside the Toolpad editor.</p>
 
-## `createComponent`
+The steps below explain how to create and use a custom component in Toolpad.
 
-Toolpad exposes a `createComponent` API to bring in custom components to the Toolpad component library.
+## Create component
 
-For example, in `toolpad/components/HelloWorld.tsx`:
+1. Start by clicking on Create in the custom component section of the Component library.
+2. Name your component.
+3. Open the snackbar prompt, it will open up the code editor. In case it doesn't open, check [troubleshoot missing editor.](https://mui.com/toolpad/how-to-guides/editor-path/)
```

Comment:
```suggestion
3. Open the snackbar prompt. It will open your code editor. In case it doesn't open, check [troubleshoot missing editor.](https://mui.com/toolpad/how-to-guides/editor-path/)
```

Reply:
Also we definitely should implement a more visible way for the user to start working on their new code component right away.